### PR TITLE
Dashboard slider

### PR DESCRIPTION
**Pull request by mazsam**
- [x] Create Slider on Dashboard. github issue #18 
slider

### CODE REVIEW

File: src/containers/Dashboard/index.js [hunk 1]
```diff
@@ -30,9 +32,27 @@ class Dashboard extends Component {
...
-                        <Text>Temporary Log out here!</Text>
-                    </TouchableOpacity>
+                    
+                    <Swiper style={style1.wrapper}
+                        dot={<View style={{backgroundColor: '#d4d2cf', width: 11, height: 11, borderRadius: 5, marginLeft: 3, marginRight: 3, marginTop: 3, marginBottom: 3}} />}
+                        activeDot={<View style={{backgroundColor: '#db4b35', width: 11, height: 11, borderRadius: 5, marginLeft: 3, marginRight: 3, marginTop: 3, marginBottom: 3}} />}
+                        paginationStyle={{ right: null, left:40}} loop>
+                        <View style={style1.slide1}>
+                            <Image resizeMode='stretch' style={styles.image} source={require('../../assets/images/kurir-1.jpg')} />
+                            <Text style={style1.text}>Compettitive Rates</Text>
```
Comment (kru) [inline src/containers/Dashboard/index.js]: semua assets di import dari satu file mas `assets/index.js`, coba liat di register dan login implementasinya, jgn langsung di require biar rapi.

Comment (mazsam) [reply src/containers/Dashboard/index.js]: oke mas kris

File: src/containers/Dashboard/index.js [hunk 2]
```diff
@@ -30,9 +32,27 @@ class Dashboard extends Component {
...
+                            <Text style={style1.subtext}>World class services</Text>
+                        </View>
+                        <View style={style1.slide1}>
+                            <Image resizeMode='stretch' style={styles.image} source={require('../../assets/images/kurir-2.jpg')} />
+                            <Text style={style1.text}>Compettitive Rates 2</Text>
```
Comment (kru) [inline src/containers/Dashboard/index.js]: Compettitive -> Competitive

**Comment by kru** [comment]
Looks good. Thank you @mazsam

**Pull request merged**